client/src/app/site/agenda/components/agenda-list/agenda-list.component.ts

```diff
@@ -341,4 +342,29 @@ export class AgendaListComponent extends BaseListViewComponent<ViewItem> impleme
             return result;
         }
     }
+
+    /**
+     * Duplicates a single selected item.
+     *
+     * @param item The item to duplicte.
+     */
+    public duplicateSingleItem(item: ViewItem): void {
```

Comment:
You just want to duplicate topics, so rename this function and use `Topic` as the parametertype.

Reply:
`Topic` as parametertype is not possible, since `item` is of type `ViewItem`.

Reply:
But what happens, if you pass an item with a motion block as a content object into this function? This function is more a `duplicateTopic` (and may be moved to the topic repository), beauce it behaves wrong on items wthout topics as content objects.